.clang-format

```diff
@@ -6,6 +6,9 @@ IndentWidth: 8
 UseTab: Always
 
 AccessModifierOffset: -8
+AlignAfterOpenBracket: Align
 AlwaysBreakAfterReturnType: TopLevel
+ContinuationIndentWidth: 8
+SpaceAfterTemplateKeyword: false
```

Comment:
I'm not sure about this option.
In `Dec.hpp` and `Traits.hpp`, we have a space after `template`.

Reply:
Dropped this change, didn't notice it sorry.